float64 centerX
float64 centerY
int32 radius
